src/material-ui-stories/Progress.stories.tsx

```diff
@@ -219,11 +230,13 @@ export const LinearDeterminate = () => {
   );
 };
 
-LinearDeterminate.parameters = {
+LinearDeterminateInternal.parameters = {
```

Comment:
parameters should be applied to the external story itself. Currently, it breaks ignoring this story in chromatic.